src/db/spanner/models.rs

```diff
@@ -1552,4 +1552,11 @@ impl Db for SpannerDb {
     fn clear_coll_cache(&self) {
         self.coll_cache.clear();
     }
+
+    fn check(&self) -> Result<bool> {
+        // TODO: is there a better check than just fetching UTC?
+        self.sql("SELECT UNIX_SECONDS(CURRENT_TIMESTAMP())")?
+            .execute(&self.conn)?;
```

Comment:
I suppose you could also call `.one()?` here just to be strict.

nit: `UNIX_SECONDS` conversion isn't really needed

Reply:
Yeah, the original idea was to do the same as mysql and track the number of seconds that things were up. Kinda useless now, though.